src/main/java/com/cloudogu/wiki/WikiDispatcherServlet.java

```diff
@@ -5,6 +5,8 @@
  */
 package com.cloudogu.wiki;
 
+import com.cloudogu.wiki.scmm.ScmWikiProvider;
+import com.cloudogu.wiki.scmm.SessionCacheScmWikiListStrategy;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.cloudogu.wiki.scmm.SessionCacheScmWikiListStrategy'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)